testsuite/features/qam/reposync/srv_sync_qam_products.feature

```diff
@@ -126,3 +126,11 @@ Feature: Synchronize products in the products page of the Setup Wizard
     Then I should see the "SUSE Manager Proxy 4.1 x86_64" selected
     When I click the Add Product button
     And I wait until I see "SUSE Manager Proxy 4.1 x86_64" product has been added
+
+  Scenario: SUSE Manager Retail Branch Server 4.1 x86_64
```

Comment:
Fine.. just to have a mirror of 4.1 for next 4.2.
But ofc, this will not apply to Uyuni

Reply:
We could also switch to 4.2 here - it's head branch after all. And it will soon come to existence as we are at alpha 1.

Tell me if you prefer that.

Reply:
We need to have here whatever "Head" version shown in the WebUI, as it shows 4.2, we should have here 4.2.
BUT keep in mind that we will not execute this code either for Uyuni or Head. As the purpose of this code is only for Maintenance Updates.

Reply:
I know that. The thing is, this code will eventually become the 4.2 QAM code.

I'll change 4.1 to 4.2 in head.

Reply:
Done.